xcm/src/v0/traits.rs

```diff
@@ -43,10 +46,15 @@ pub enum Error {
 	FailedToDecode,
 	BadOrigin,
 	ExceedsMaxMessageSize,
+	/// An asset transaction (like withdraw or deposit) failed.
+	/// See implementors of the `TransactAsset` trait for sources.
```

Comment:
spotted by the spellcheck CI check
```suggestion
	/// See implementers of the `TransactAsset` trait for sources.
```